build/packages-template/bbb-config/after-install.sh

```diff
@@ -130,7 +130,7 @@ if [ -d /var/mediasoup/screenshare ]; then
 fi
 
 sed -i 's/worker_connections 768/worker_connections 4000/g' /etc/nginx/nginx.conf
-
+sed -i '/include \/etc\/nginx\/conf\.d\/\*\.conf\;/ a \\tlimit_conn_zone $uri zone=ws_zone:5m;' /etc/nginx/nginx.conf
```

Comment:
In order to keep it easy to maintain, I propose that you do it like this:
```
echo 'limit_conn_zone $uri zone=ws_zone:5m;' > /etc/nginx/conf.d/html5-conn-limit.conf
```

PS: I haven't tested, please check if it works.